pkgs/by-name/rk/rkbin/package.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rkbin";
+  version = "2023.07.26-${gitShort}";
+
+  gitCommit = "b4558da0860ca48bf1a571dd33ccba580b9abe23";
+  gitShort = builtins.substring 0 7 gitCommit;
+
+  src = fetchFromGitHub {
+    owner = "rockchip-linux";
+    repo = "rkbin";
+    rev = "${gitCommit}";
+    hash = "sha256-KUZQaQ+IZ0OynawlYGW99QGAOmOrGt2CZidI3NTxFw8=";
+  };
+
+  installPhase = ''
+    mkdir $out
+    mv bin doc $out/
+  '';
+
+  ROCKCHIP_TPL_RK3588 = "bin/rk35/rk3588_ddr_lp4_2112MHz_lp5_2736MHz_v1.12.bin";
```

Comment:
Do you need to compile a different rkbin, depending on the board type? Or why does this encode a specific board type here?

Reply:
But does this make this `rkbin` one that's only applicable for rk3588?